### PR TITLE
Sixth chunk of terrain descriptions

### PR DESCRIPTION
**Pull request by DracoGriffin**
Another round of descriptions to be reviewed.

Some descriptions are a bit obtuse since they are constructed by player and the construction menu goes by roofs, not floors -- additionally as floors and roofs are treated as the same terrain by code.

Specific vocabulary is used for players to more readily identify how terrain may be interacted with (example would be examine, as some terrains can be interacted through examine hotkey).

Furthermore, descriptions may not always be 100% appropriate as different locations reference terrain in different ways (pavements, sidewalks and such -- especially "walkway" and "metal grate" aren't referenced properly in maps so did my best to describe more broadly).

Side notes:

I tried to find a way to do snippet categories for the stained glasses, but I couldn't get it to work (if it's even possible). Similar to the way [other items have variant descriptions for the same item](https://github.com/CleverRaven/Cataclysm-DDA/blob/75cb9e7e0a0889298f57be5994484f8f348ea44b/data/json/items/armor.json#L5815).

### CODE REVIEW

**Comment by illi-kun** [comment]
Jenkins rebuild

File: data/json/terrain.json [hunk 1]
```diff
@@ -2813,6 +2822,7 @@
         "type" : "terrain",
         "id" : "t_window_reinforced_noglass",
         "name": "reinforced boarded up window",
+        "description": "A heavily fortified glass window with carefully placed planks across the open frame to block vision.  Adding a few spikes and metal plating would further increase its durability.",
```
Comment (KongMD-Steam) [inline data/json/terrain.json]: Minor nitpick: remove 'glass' from the description, as this terrain doesn't have glass.

File: data/json/terrain.json [hunk 2]
```diff
@@ -2980,6 +2998,7 @@
         "type" : "terrain",
         "id" : "t_rock_smooth",
         "name": "smoothed rock",
+        "description": "Shaped naturally occurring stonework, commonly granite or marble for older structures.",
```
Comment (Leland) [inline data/json/terrain.json]: I'm not entirely sure what this means...


> Shaped naturally occurring stonework, commonly granite or marble for older structures

Should the first part read something along the lines of, "A block of stone that's been smoothed and shaped"?

And what does the second part mean? Smoothed granite and marble are more common in older structures than other types of smoothed rock?

File: data/json/terrain.json [hunk 3]
```diff
@@ -2886,6 +2899,7 @@
         "type" : "terrain",
         "id" : "t_window_bars_alarm",
         "name": "window with metal bars",
+        "description": "A giant sheet of glass inserted into a window with thick security grilles, making it impossible to crawl through.  Typically installed for high-value stores, or at least stores in bad neighborhoods.  This one has a small sticker in a corner stating, 'Protected by AtmoWeb, leading AI in terminating crime'.",
```
Comment (Leland) [inline data/json/terrain.json]: > Protected by AtmoWeb, leading AI in terminating crime

I like what you've done here, but this is definitely misleading, right? Doesn't this just ring an alarm when breached? "Terminating crime" seems a bit extreme

File: data/json/terrain.json [hunk 4]
```diff
@@ -2736,6 +2741,7 @@
         "type" : "terrain",
         "id" : "t_window_frame",
         "name": "window frame",
+        "description": "A wooden window frame that has shattered glass around it.  Some of those shards look jagged and harmful if you tried to climb through.  You could smash the rest out loudly, or you could clean the window of the broken pieces quietly but it'd take longer.",
```
Comment (Leland) [inline data/json/terrain.json]: > Some of those shards look jagged and harmful if you tried to climb through

Nit: consider rewording. They'd be jagged and harmful even if you didn't try to climb through.

> You could smash the rest out loudly, or you could clean the window of the broken pieces quietly but it'd take longer

Second phrase is quite wordy, consider rewording. Maybe something like:

> You could smash out the remaining pieces, or take your time and quietly clean them up.

File: data/json/terrain.json [hunk 5]
```diff
@@ -2714,6 +2718,7 @@
         "type" : "terrain",
         "id" : "t_window_empty",
         "name": "empty window",
+        "description": "An empty window frame bordered with two by fours and nails.  A sheet of glass could be installed, or even boarded up for protection.  Also useful for converting into a wall.",
```
[the remark this answers is not in the document]
Comment (DracoGriffin) [reply data/json/terrain.json]: This is one of those weird terrains that have "multi-functions" or rather, not referenced appropriately.

t_window_empty can be a step in building wood walls:
```json
    },{
        "type" : "construction",
        "description" : "Build Wood Wall", "//": "Step 1: half of a wall where a window used to be",
        "category" : "CONSTRUCT",
        "required_skills" : [ [ "fabrication", 2 ] ],
        "time" : 50,
        "qualities": [ [
            { "id": "HAMMER", "level": 2 }
        ] ],
        "components" : [
            [ [ "2x4", 10 ] ],
            [ [ "nail", 20 ] ]
        ],
        "pre_terrain" : "t_window_empty",
        "post_terrain" : "t_wall_half"
    },{
```

File: data/json/terrain.json (hunk 1 above)
Comment (DracoGriffin) [reply data/json/terrain.json]: Good catch, thanks

File: data/json/terrain.json (hunk 3 above)
Comment (DracoGriffin) [reply data/json/terrain.json]: This is just meant to give a tiny hint to anyone that takes the time to read the description that this one is alarmed (even in real life, sometimes the best deterrent is a security sticker). Secondly, it's meant to be an easter egg/reference to Terminator series.

Atmo = Sky, Web = Net, leading _AI_ in _terminating_ crime

And I mean, the broken window triggers Eyebots, which then summon security bots (which can be lethal if you're not able to handle them) if you're spotted.

File: data/json/terrain.json (hunk 2 above)
Comment (DracoGriffin) [reply data/json/terrain.json]: This is actually a stilted sentence, that I should've rewrote. However, this terrain is referenced only in two very different places: [basements](https://github.com/CleverRaven/Cataclysm-DDA/blob/93f2979292e8e0082ea32f6a9720ef51d810b8e8/src/mapgen_functions.cpp#L2932) and [cemeteries](https://github.com/CleverRaven/Cataclysm-DDA/blob/3f93d321a11aae1850cbfeabc2314ff08bffbcb2/data/json/mapgen/cemetery_4square.json#L91). Hence one of the reasons I write in the PR intro that trying to write some of these descriptions can be very broad.

So for whatever reason, the cemetery looks like where they hold a wake or funeral but in a building of smoothed rock? So, I researched what type of buildings like that would be found in New England region and granite/marble were the most common. The first part is meant more for the basements.

I think your `A block of stone that's been smoothed and shaped` is a good start that I'll work off of.

**Comment by DracoGriffin** [comment]
Updated per comments.

**Pull request merged**